src/main/java/solutions/fairdata/fdp/index/web/controller/HomeController.java

```diff
@@ -37,8 +39,14 @@
     private IndexEntryService service;
 
     @GetMapping
-    public String home(Model model, Pageable pageable) {
+    public String home(Model model, @SortDefault(sort = "modificationTime", direction = Sort.Direction.DESC) Pageable pageable) {
+        String sort = "";
+        if (!pageable.getSort().toList().isEmpty()) {
+            sort = pageable.getSort().toList().get(0).getProperty() + "," + pageable.getSort().toList().get(0).getDirection().name().toLowerCase();
```

Comment:
These chains can be a bit difficult in terms of readability and maintenance. Maybe the list can be stored as an intermediate variable?

```
var list = pageable.getSort().toList();
if (!list.isEmpty() {
    var x = list.get(0); // any chance this is null? otherwise can be wrapped in an Optional
    sort = x.getProperty() + "," + x.getDirection().getName().toLowerCase();
}
```

Reply:
I tried to improve it using stream so even no "if" anymore 😄